Add a LICENSE file with the same license as perl has

bitstreamout opened the pull request:
Signed-off-by: Werner Fink werner@suse.de

bitstreamout committed:
Add a LICENSE file with the same license as perl has

Signed-off-by: Werner Fink <werner@suse.de>

MidLifeXis commented:
See also the meta spec: http://search.cpan.org/dist/CPAN-Meta/lib/CPAN/Meta/Spec.pm#license

bitstreamout committed:
Second try: Add a License with the same license as perl has

For this second version I've used the module ExtUtils::MakeMaker
and the various descriptions about this module.

Signed-off-by: Werner Fink <werner@suse.de>
